examples/ct/ct_server/main.go

```diff
@@ -35,6 +35,9 @@ import (
 	"google.golang.org/grpc/naming"
 )
 
+// config represents all configurable settings of {CT,Log,Map} server and Log signer.
```

Comment:
nit: Replace "{CT,Log,Map} server and Log signer" with just "CT server". This particular config is exclusive to CT server (and so are the others to their respective binaries).

Ditto for others.